src/pages/dashboard/settings/Plans.vue

```diff
@@ -146,6 +147,32 @@ const hightLights = computed<Stat[]>(() => ([
 
 <template>
   <div v-if="!isLoading" class="h-full bg-white max-h-fit dark:bg-gray-800">
+
+      <div class="px-4 pt-6 mx-auto max-w-7xl lg:px-8 sm:px-6" v-if="!displayPlans && currentPlan?.name !== 'free'">
```

Comment:
Why the check for displayPlans?

Reply:
Yeah, this change prevents this PR from working. Without it works great, but with it I cannot get this new div to show